include/deal.II/base/bounding_box.h

```diff
@@ -134,18 +136,27 @@ class BoundingBox
    */
   double volume() const;
 
+  /**
+   * Boost serialization function
+   */
+  template<class Archive>
+  void serialize(Archive &ar, const unsigned int version)
```

Comment:
You will want to avoid the warning about the unused argument by writing this as
```
void serialize(Archive &ar, 
                      /*const unsigned int version*/)
```